### ISSUE TITLE
`version_option` displays click-extra version

### ISSUE THREAD

**Issue by rasa** [kdeldycke/click-extra#176]
Using click-extra 2.1.1, when I run:
```python
import click

@click.command()
@click.version_option(version="0.1")
def cli():
    pass

if __name__ == "__main__":
    cli()
```
I get:
```console
test.py, version 0.1
```
When I run:
```python
import click_extra as click

@click.command()
@click.version_option(version="0.1")
def cli():
    pass

if __name__ == "__main__":
    cli()
```
I get:
```console
test-extra-version.py, version 2.1.1
```
But hey, it's in color! :)

**Comment by kdeldycke** [comment]
> But hey, it's in color! :)

Aha! 😅

Yes definitely a bug.

**Comment by kdeldycke** [comment]
I have a hard time finding a solution for this one so in the mean time I've reproduced that bug in the form of a unittest: https://github.com/kdeldycke/click-extra/commit/c8cc30b3e21ae7ac9758bdf0004dddaf8cb3f33f

**Comment by rasa** [comment]
This has been fixed in 3.0.1

**Comment by kdeldycke** [comment]
> This has been fixed in 3.0.1

Oh great! I suspected my hard-copy of Click's code would have an effect on it and planned to revisit this issue. Thanks for doing this for me and confirming the fix!

**Comment by kdeldycke** [comment]
Note this might be related to: https://github.com/pallets/click/issues/2331
